### PR TITLE
getting data.data.data from youtube

### PR DESCRIPTION
**Pull request by sveranicola**

### CODE REVIEW

**Review by GriffGeorgiadis — DISMISSED** [review]
🇺🇸 🦅

File: server/controllers/Oauth.js [hunk 1]
```diff
@@ -2,7 +2,10 @@ const Oauth = require('../../database/models/Oauth.js');
 
 module.exports = {
   newUser: (req, res) => {
+<<<<<<< HEAD
+=======
     console.log(typeof req.params)
```
Comment (bakese) [inline server/controllers/Oauth.js:7]: remove console.log

File: server/controllers/Oauth.js [hunk 2]
```diff
@@ -2,7 +2,10 @@ const Oauth = require('../../database/models/Oauth.js');
 
 module.exports = {
   newUser: (req, res) => {
+<<<<<<< HEAD
```
Comment (bakese) [inline server/controllers/Oauth.js:5]: remove Head stuff

File: src/components/feed/Feed.jsx [hunk 3]
```diff
@@ -34,8 +35,21 @@ var Feed = ({ setIsGoogleSignedIn }) => {
   useEffect(() => {
     signIn()
     setIsGoogleSignedIn(true);
+
+    if( localStorage.access_token) {
+      axios.get(`/getYoutube/${localStorage.access_token}`)
+        .then(data => {
+          console.log(data);
```
Comment (bakese) [inline src/components/feed/Feed.jsx:42]: remove console.log

**Review by bakese — COMMENTED** [review]
remove console.logs and head

**Pull request closed without merging**